core/io/resource_format_binary.cpp

```diff
@@ -79,6 +79,8 @@ enum {
 	VARIANT_VECTOR2I = 45,
 	VARIANT_RECT2I = 46,
 	VARIANT_VECTOR3I = 47,
+	VARIANT_INT64_ARRAY = 48,
```

Comment:
There's `VARIANT_DOUBLE` above, not sure why it wasn't `VARIANT_REAL` but should it be renamed `VARIANT_FLOAT` too?

Reply:
There's also `VARIANT_RAW_ARRAY` which could be `VARIANT_BYTE_ARRAY` I guess.